salt/common/tools/sbin/so-status

```diff
@@ -290,22 +291,29 @@ main() {
 
 # {% endraw %}
 
+is_tty() {
+    __tty=0
+    [ -t 1 ] \
+        && __tty=1
+    # don't print colors if NO_COLOR is set to anything
+    [ "${#NO_COLOR}" -ne 0 ] \
+        && __tty=0
```

Comment:
Why do you add a line break here and on L296-297?

Reply:
Personal style/preference. I find it easier to read as two lines as opposed to just one.

Reply:
Let's stay consistent with the rest of Security Onion and keep these on one line.